### DISCUSSION TITLE
no base url in sitemap.xml in netlify

### DISCUSSION THREAD

**Discussion by rezagray** [daattali/beautiful-jekyll#931] [Q&A]
hello.
i have problem with sitemap in netlify. the sitemap generator didn't add my base url.
![image](https://user-images.githubusercontent.com/53551226/151330190-350eead2-1c5e-4ea0-9c11-733b5f3d7b0b.png)

in local its working normally
![image](https://user-images.githubusercontent.com/53551226/151330263-765841df-bb52-4e24-a162-ab9a3495669f.png)
in github pages its working normally
![image](https://user-images.githubusercontent.com/53551226/151330581-b7dce417-cb29-4f98-b377-c3cc73c2feb2.png)
in netlify preview mode its also working normally.

is there any solution?

**Comment by aisbergde** [comment]
I have the same issue.
the generated sitemap doesn't match the requirements:

https://developers.google.com/search/docs/crawling-indexing/sitemaps/build-sitemap?hl=en

Referenced URLs' properties: **Use fully-qualified, absolute URLs in your sitemaps.** Google will attempt to crawl your URLs exactly as listed. For example, if your site is at https://www.example.com/, don't specify a URL such as /mypage.html (a relative URL), use the complete, absolute URL: https://www.example.com/mypage.html.

https://www.sitemaps.org/protocol.html#location

How to get a valid sitemap, containing fully-qualified, absolute URLs?

**Comment by aisbergde** [comment]
I found the solution:
https://github.com/jekyll/jekyll-sitemap

```
url: "https://example.com" # the base hostname & protocol for your site
plugins:
  - jekyll-sitemap

```
the `url: ...` was missing in my `_config.yml`
